verifier/trustpolicy/oci.go

```diff
@@ -25,17 +25,17 @@ import (
 	"github.com/notaryproject/notation-go/internal/trustpolicy"
 )
 
-// OCIDocument represents a trustPolicy.json document for OCI artifacts
-type OCIDocument struct {
+// Document represents a trustpolicy.json document
+type Document struct {
 	// Version of the policy document
 	Version string `json:"version"`
 
 	// TrustPolicies include each policy statement
-	TrustPolicies []OCITrustPolicy `json:"trustPolicies"`
+	TrustPolicies []TrustPolicy `json:"trustPolicies"`
 }
 
-// OCITrustPolicy represents a policy statement in the policy document for OCI artifacts
-type OCITrustPolicy struct {
+// TrustPolicy represents a policy statement in the policy document
+type TrustPolicy struct {
```

Comment:
The order of the fields was changed by #394

Reply:
This change can be treated as a breaking change. Should be fixed in both this PR and the `main` branch.

Reply:
@shizhMSFT Oh, that's a good catch. 
I guess it's a breaking change due to the way to initialize a struct in Golang? i.e., they could just follow the field order, without the name, while generating a new struct.